src/websockets/legacy/compatibility.py

```diff
@@ -22,3 +22,12 @@ def loop_if_py_lt_38(loop: asyncio.AbstractEventLoop) -> Dict[str, Any]:
 
         """
         return {"loop": loop}
+
+
+if sys.version_info[:2] < (3, 11):
+    from .async_timeout import timeout as asyncio_timeout  # pragma: no cover
+else:  # pragma: no cover
+    from asyncio import timeout as asyncio_timeout
+
+
+__all__ = ["asyncio_timeout", "loop_if_py_lt_38"]
```

Comment:
This isn't needed because aren't doing `from websockets.legacy.compatibility import *` anywhere.

Did I miss a reason why it would be useful? Or did you just add it for consistency?

Reply:
Ah, it avoids this warning from mypy:

src/websockets/legacy/protocol.py:56: error: Module "websockets.legacy.compatibility" does not explicitly export attribute "asyncio_timeout"  [attr-defined]